### DISCUSSION TITLE
Questions about support class cache for ByteBuddy

### DISCUSSION THREAD

**Discussion by zhangzhen108** [apache/skywalking#9136] [Q&A]
hello，I have a question about #4858  to ask you。
 If other agents do not enhance the class during the first load (transform), the class will be enhanced during the second transform. Then skywalking directly returns the byte stream cached for the first time in the second time, will it overwrite the content enhanced for the second time by other agents? Because the order in which the transform is executed depends on the order in which the Agent is configured.

**Comment by Superskyyy** [comment]
@zhangzhen108 Please do not duplicate your discussion in multiple locations (You just posted the exact same questions in 3 locations), it becomes noise to whoever is watching the repository activities.

**Comment by wu-sheng** [comment]
Pinging me in many threads is really very annoying. That PR is not written by me, and it carried a plugin test to verify the process.
I don't understand why you pinged me. I have been very clear, I am not knowing other agents at all, also supporting how other agents work is not my or skywalking community responsibility.
